index.bs

```diff
@@ -495,6 +506,30 @@ through a process called [=sensor fusion=].
 [=calibration|Calibrated=] [=raw sensor readings=] are referred to as <dfn>sensor readings</dfn>,
 whether or not they have undergone [=sensor fusion=].
 
+If the [=sensor readings=] represent spatial measurements (e.g.
+acceleration, angular velocity), they must be resolved in
+a <dfn export>local coordinate system</dfn> that represents
+a reference frame for these [=sensor readings|readings=].
+A [=device sensor|sensor=] that provides such [=sensor readings|readings=]
+is referred to as <dfn export>spatial sensor</dfn>.
+
+Depending on the number of orthogonal axes in which the measurements
```

Comment:
I find this part a bit hard to read... very long sentence with injected sentences, which then continues

Reply:
Wrapping it around makes it easier to read

A [=spatial sensor|sensor=] can be <dfn>uniaxial</dfn>, <dfn>biaxial</dfn>, or <dfn>triaxial</dfn>, depending on the number of orthogonal axes in which the measurements can be performed simultaneously by a [=spatial sensor=]

Reply:
A [=spatial sensor=] can be <dfn>uniaxial</dfn>, <dfn>biaxial</dfn>,
or <dfn>triaxial</dfn>, depending on the number of orthogonal axes in
which it can perform simultaneous measurements.

Reply:
fixed in the latest patch